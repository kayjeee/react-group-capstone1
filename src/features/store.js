import { configureStore } from '@reduxjs/toolkit';
import rocketReducer from './rockets/rocketSlice';
import missionReducer from './missions/missionsSlice';

const store = configureStore({
  reducer: {
    rockets: rocketReducer,
    missions: missionReducer,
  },
});

export default store;
