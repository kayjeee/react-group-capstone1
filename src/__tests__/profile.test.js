import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import React from 'react';
import MyProfile from '../components/myProfile';
import store from '../features/store';

describe('User Interactions test', () => {
  it('will test user interaction', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <MyProfile />
        </Provider>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
