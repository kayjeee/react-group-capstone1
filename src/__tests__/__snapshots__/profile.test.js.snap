// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Interactions test will test user interaction 1`] = `
<section
  className="profile-section"
>
  <div
    className="missions"
  >
    <h2>
      My Missions
    </h2>
    <ul
      className="profile-list"
    />
  </div>
  <div
    className="missions"
  >
    <h2>
      My Rockets
    </h2>
    <ul
      className="profile-list"
    />
  </div>
</section>
`;
